sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -212,6 +212,7 @@ export class ManagementClient extends LinkEntity {
     throwErrorIfConnectionClosed(this._context.namespace);
     try {
       if (!this._isMgmtRequestResponseLinkOpen()) {
+        await this._context.namespace.readyToOpenLink();
```

Comment:
Just realized I can move this check into `_negotiateClaim()` so it only needs to be done in one place, versus every client that calls `_negotiateClaim()`. Will update.

Reply:
Updated and running test one more time for sanity.